administrator/language/de-DE/com_tags.ini

```
@@ -42,8 +42,7 @@ COM_TAGS_EMPTYSTATE_BUTTON_ADD="Erstes Schlagwort erstellen"
 COM_TAGS_EMPTYSTATE_CONTENT="Schlagworte in Joomla! bieten eine flexible Möglichkeit, um Inhalte zu organisieren. Das gleiche Schlagwort kann auf viele verschiedene Einträge in verschiedenen Inhaltstypen angewendet werden."
 COM_TAGS_EMPTYSTATE_TITLE="Es wurden noch keine Schlagwörter erstellt."
 COM_TAGS_DELETE_NOT_ALLOWED="Das Löschen des Schlagworts „%s“ ist nicht erlaubt."
-COM_TAGS_ERROR_UNIQUE_ALIAS="Ein anderes Schlagwort hat denselben Alias."
-COM_TAGS_ERROR_UNIQUE_ALIAS_TRASHED="Ein anderes Schlagwort hat denselben Alias. Dieses Schlagwort befindet sich im Papierkorb."
+COM_TAGS_ERROR_UNIQUE_ALIAS="Ein anderes Schlagwort hat denselben Alias (ggf. ist es im Papierkorb)."
```

```suggestion
COM_TAGS_ERROR_UNIQUE_ALIAS="Ein anderes Schlagwort hat denselben Alias."
COM_TAGS_ERROR_UNIQUE_ALIAS_TRASHED="Ein anderes Schlagwort hat denselben Alias. Dieses Schlagwort befindet sich im Papierkorb."
```

Included in #2647 merged in 4.2